WBooks/BookDB.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  BookDB.swift
+//  WBooks
+//
+//  Created by Matías David Schwalb on 05/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import Foundation
+
+struct BookDB {
```

Comment:
Why do you need this class?

Reply:
It holds all the books in it's property bookArrayDB so there is only one instance holding this information. This prevents unnecessary copying when accessing different views, any class who wants to access certain book just needs it's bookID.

Reply:
Okay, in that case, it should be in the corresponding View Model. Let's leave it like that for the time being. We'll do it once we get to the Refactor card.